hips/hip-0012.md

```diff
@@ -193,4 +365,12 @@ It is in the community's best interest for the developers to move at a deliberat
 ## References
 
 - The Helm org's [governance documents](https://github.com/helm/community/tree/master/governance)
-- The [Helm 3 planning documents](https://github.com/helm/community/tree/master/hips/archives/helm/helm-v3)
\ No newline at end of file
+- The [Helm 3 planning documents](https://github.com/helm/community/tree/master/hips/archives/helm/helm-v3)
+
+## Appendix
+
+### Helm 4 proposal template
+
+```
+TBD
+```
```

Comment:
How should this be different from a HIP?

Reply:
(will discuss in slack)